python/kernmlops/kernmlops_benchmark/mongodb.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+from dataclasses import dataclass
+from typing import cast
+
+from data_schema import GraphEngine, demote
+from kernmlops_benchmark.benchmark import Benchmark, GenericBenchmarkConfig
+from kernmlops_benchmark.errors import (
+  BenchmarkNotInCollectionData,
+  BenchmarkNotRunningError,
+  BenchmarkRunningError,
+)
+from kernmlops_config import ConfigBase
+
+
+@dataclass(frozen=True)
+class MongoDbConfig(ConfigBase):
+  record_count: int = 1000000
+  readProportion: float = 0.25
```

Comment:
please also fix these variable names

Reply:
please change these names to `read_proportion` and `update_proportion` then re-generate the `defaults.yaml` with `make defaults`

Reply:
this is not resolved, please change these names to read_proportion and update_proportion then re-generate the defaults.yaml with make defaults